modules/mobile/spec/request/prescriptions_request_spec.rb

```diff
@@ -0,0 +1,255 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/rx_client_helpers'
+require 'support/shared_examples_for_mhv'
+require_relative '../support/iam_session_helper'
+require_relative '../support/matchers/json_schema_matcher'
+
+RSpec.describe 'health/rx/prescriptions', type: :request do
+  include JsonSchemaMatchers
+  include Rx::ClientHelpers
+
+  let(:mhv_account_type) { 'Premium' }
+  let(:json_body_headers) { { 'Content-Type' => 'application/json', 'Accept' => 'application/json' } }
+
+  before(:all) do
+    @original_cassette_dir = VCR.configure(&:cassette_library_dir)
+    VCR.configure { |c| c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes' }
+  end
+
+  after(:all) { VCR.configure { |c| c.cassette_library_dir = @original_cassette_dir } }
+
+  before do
+    allow_any_instance_of(MHVAccountTypeService).to receive(:mhv_account_type).and_return(mhv_account_type)
+    allow(Rx::Client).to receive(:new).and_return(authenticated_client)
+    current_user = build(:iam_user, :mhv)
+
+    iam_sign_in(current_user)
+  end
+
+  describe 'GET /mobile/v0/health/rx/prescriptions' do
+    context 'with a valid EVSS response and no failed facilities' do
+      it 'returns 200', :aggregate_failures do
```

Comment:
I believe you can add `:aggregate_failures` to the describe block and you'll get it for all the tests. Frankly, I wonder if we should just set it in our Rspec config.